app/script.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+$(function() {
+
+  //flags for word lock feature
+  var adjLocked = false;
+  var verbLocked = false;
+  var nounLocked = false;
```

Comment:
this is a great use case for flags like this. Why even make the call if the word is locked?